lib/ws-kernel.coffee

```diff
@@ -18,8 +18,14 @@ class WSKernel extends Kernel
     interrupt: ->
         @session.kernel.interrupt()
 
+    shutdown: (restart = false) ->
+        if restart
+            return @session.kernel.restart()
+        else
+            return @session.kernel.shutdown()
+
     restart: ->
-        @session.kernel.restart()
+        return @session.kernel.restart()
```

Comment:
I'm aware (though this came as a surprise to me, because I don't think hydrogen coding style generally uses implicit returns)